api_backend.yaml

```diff
@@ -1010,6 +1032,26 @@ definitions:
           - name
           - family_name
       - $ref: "#/definitions/LimitedFederatedUser"
+  TokenType:
+    type: string
+    description: The type of the provided access token.
+    x-extensible-enum:
+      - Bearer
+  SupportToken:
+    title: Support token
+    description: A Support Token response
+    type: object
+    properties:
+      access_token:
+        type: string
+      expires_in:
```

Comment:
what's this value? where's the start date saved? can it be expires_at with a date instead?

Reply:
Commonly the use of `expire_in` is preferred because doesn't need clock synchronization to verify (client-side) when the token expires.

Reply:
if the access_token is a JWT, doesn't it contains the `exp` claim itself?

Reply:
Yes it contains `exp` claim, that contains the Date (in seconds) of token expiration, while `expires_in` contains a time expression of token validity. For example if we generate a support token right now and the TTL for this token is `7 days` we have:
- `exp` contains the date expressed in seconds for next week;
- `expires_in` contains `604800` (the number of seconds in 7 days)

Reply:
Yes, it does. The `exp` claim is a timestamp and could be extracted with token introspection. At server level clock synchronization is required and we can use `clockTolerance` for configuring the clock screw. `expire_in` add more information only on client side.